website/src/blogs/2018-09-27-celebrate-hacktoberfest-2018-with-openebs.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Celebrate Hacktoberfest 2018 with OpenEBS !!
+author: Uma Mukkara
+author_info: Contributor at openebs.io, Co-founder & COO@MayaData. Uma led product development in the early days of MayaData (CloudByte).
+date: 27-09-2018
+tags: Digital Ocean, Hacktoberfest 2018, Hacktoberfest, OpenEBS, Open Source
+excerpt: The 5th edition of Hacktoberfest is around the corner and it will be the second Hacktoberfest experience for the OpenEBS community. All of us are excited!
+---
+
+The 5th edition of [Hacktoberfest](https://hacktoberfest.digitalocean.com/) is around the corner and it will be the second [Hacktoberfest](https://openebs.io/hackfests/hacktoberfest-2018) experience for the OpenEBS community. All of us are excited!
+
+In August 2017, OpenEBS community began growing and it was a beginning to building a strong foundation for an open source project. We were first introduced to Hacktoberfest by friends & peers at the DigitalOcean Bangalore Meetup and were interested in participating in it. We enlisted OpenEBS as one of the projects participating in Hacktoberfest 2017. We were pleasantly surprised with the participation and enthusiasm that Hacktoberfest attracts from developers around the world. The result was that the OpenEBS community grew by 20+ new contributor friends with ~100 PRs.
+
+We are excited to announce that OpenEBS would be participating in Hacktoberfest 2018 as well! Look for latest updates [here](https://openebs.io/hacktoberfest)
+
+## How do you get started with Hacktoberfest?
+
+GitHub makes it really simple for contributors to get started. Just search for the tag “[hacktoberfest](https://github.com/search?l=&amp;q=state%3Aopen+label%3Ahacktoberfest&amp;ref=advsearch&amp;type=Issues&amp;utf8=%E2%9C%93)” at the GitHub level. Within a few minutes, you can filter tens of issues to start working on that which matches your interest. First timers can look out for “good first time” tag. Anyways, here are the quick steps for you if you are interested in participating.
+
+1. Register at [DigitalOcean](https://hacktoberfest.digitalocean.com/) website and provide your mailing address so that their system knows you are participating and can send you the t-shirt and stickers later. Well, we all want them, matter of pride 🙂
```

Comment:
```suggestion
1. Register at [DigitalOcean](https://hacktoberfest.digitalocean.com/) website and provide your mailing address so that their system knows you are participating and can send you the t-shirt and stickers later. Well, we all want them, a matter of pride 🙂
```